run_tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
With this pointing at `/bin/sh` I had an issue with environmental variables and it couldn't find the `source` command, @thbenschop leaning heavily on you for advice here..

Reply:
sh is the primitive low level shell without paths etc, this should indeed be bash